pipeline/06-upload.R

```diff
@@ -257,7 +257,11 @@ if (upload_enable) {
 
   # If values were uploaded, trigger a Glue crawler to find any new partitions
   glue_srv <- paws.analytics::glue()
-  glue_srv$start_crawler("ccao-model-results-crawler")
+  tryCatch(
+    glue_srv$start_crawler("ccao-model-results-crawler"),
+    error = function(e) message(e),
+    warning = function(w) message(e)
+  )
```

Comment:
This was added to prevent the pipeline from throwing an error if it tries to run the crawler while one is already going.